gui/src/ScenarioViewer.tsx

```diff
@@ -0,0 +1,539 @@
+        pipelines,
+        authorizedTags,
+    ] = scenario;
+
+    useEffect(() => {
+        setTags(scenario ? scenarioTags : []);
+        setProperties(
+            scenario
+                ? scenarioProperties.map(([k, v], i) => ({
+                      id: i + "",
+                      key: k,
+                      value: v,
+                  }))
+                : []
+        );
+    }, []);
+
+    return (
+        <div>
+            <Box sx={MainBoxSx}>
+                <Accordion>
+                    <MuiAccordionSummary id={`panel-${scenarioId}`}>
+                        <Grid
+                            container
+                            alignItems="center"
+                            direction="row"
+                            flexWrap="nowrap"
+                            justifyContent="space-between"
+                            spacing={1}
+                        >
+                            <Grid item>
+                                {scenarioLabel}
+                                {primary && (
+                                    <Chip
+                                        color="primary"
+                                        label={<FlagOutlined sx={FlagSx} />}
+                                        size="small"
+                                        sx={ChipSx}
+                                    />
+                                )}
+                            </Grid>
+                            <Grid item>
+                                <IconButton sx={IconPaddingSx} onClick={updateScenario}>
+                                    <Send fontSize="small" color="info" />
+                                </IconButton>
+                            </Grid>
+                        </Grid>
+                    </MuiAccordionSummary>
+                    <AccordionDetails>
+                        <Grid container rowSpacing={2}>
+                            <Grid item xs={12} container justifyContent="space-between">
+                                <Grid item xs={4} pb={2}>
+                                    <Typography variant="subtitle2">Config ID</Typography>
+                                </Grid>
+                                <Grid item xs={8}>
+                                    <Typography variant="subtitle2">{config}</Typography>
+                                </Grid>
+                            </Grid>
+                            <Grid item xs={12} container justifyContent="space-between">
+                                <Grid item xs={4}>
+                                    <Typography variant="subtitle2">Cycle / Frequency</Typography>
+                                </Grid>
+                                <Grid item xs={8}>
+                                    <Typography variant="subtitle2">{date}</Typography>
+                                </Grid>
+                            </Grid>
+                            {action === "EDIT" && (
```

Comment:
ternary operator

Reply:
There is different rendering block with <Autocomplete> below with ternary operator. So this is different block only rendering only when it's `Edit` mode.  @FredLL-Avaiga

Reply:
But that will write false in the page when action != EDIT
So you may as well group it with the ternary operator underneath